lib/pages/list/task_edit.dart

```diff
@@ -524,11 +530,11 @@ class _TaskEditPageState extends State<TaskEditPage> {
         ),
         actions: <TextButton>[
           TextButton(
-            child: Text('Cancel'),
+            child: Text('CANCEL'),
             onPressed: () => Navigator.of(context).pop(),
           ),
           TextButton(
-            child: Text('Reset'),
+            child: Text('RESET'),
```

Comment:
Why did you capitalize that?

Reply:
Following Material design guidelines. See: https://material.io/components/buttons#text-button